### PR TITLE
Change delay to 300 seconds

### PR DESCRIPTION
**Pull request by bobi12345678**
My first pull request attempt from bobi12345678

### CODE REVIEW

File: File1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-TEst
\ No newline at end of file
+TEst
+blink=300
```
Comment (ins1234) [inline File1.txt]: Probably it is better to be 400.

Comment (bobi12345678) [reply File1.txt]: Ok

File: File1.txt [hunk 2]
```diff
@@ -1,2 +1,5 @@
 TEst
-blink=300
\ No newline at end of file
+blink=400
+Line1
+Line2
```
Comment (ins1234) [inline File1.txt]: Remove Line2!

Comment (ins1234) [inline File1.txt]: Remove Line2!

Comment (bobi12345678) [reply File1.txt]: Removing it.

**Pull request merged**